backend/tests/apps/mentorship/model/test_program.py

```
@@ -0,0 +1,74 @@
+from datetime import datetime
```

Ruff (0.14.10)

File `backend/tests/apps/mentorship/model/test_program.py` is part of an implicit namespace package. Add an `__init__.py`.

(INP001)